models/comments.model.js

```diff
@@ -10,3 +10,24 @@ exports.selectCommentsById = (article_id) => {
 		}
 	});
 };
+
+exports.InsertNewCommentById = (article_id, username, body) => {
+	const queryTxt = `
+	INSERT INTO comments
+	(article_id, author, body)
+	VALUES
+	($1, $2, $3)
+	RETURNING *;
+	`;
+  if (body.length === 0) {
+		return Promise.reject({ status: 400, msg: 'Bad request' });
+	}	
+  return db.query(queryTxt, [article_id, username, body]).then((result) => {
+		if (result.rows.length === 0) {
+			return Promise.reject({ status: 404, msg: 'Not found' });
+		} else {
+			return result.rows[0];
+		}
+	});
+};
```

Comment:
Make sure to make necessary checks here in your model that both the `username` and `body` exist before making your query.